### PR TITLE
[COST-5015] Azure disk capacity table migration

### PR DESCRIPTION
**Pull request by myersCody**
## Jira Ticket

[COST-5015](https://issues.redhat.com/browse/COST-5015)

## Description

This change will add a table to store the azure disk capacities information.

## Testing

1. Checkout Branch
2. Run the Migration
3. Query the table and see values. 

## Release Notes
- [ ] proposed release note

```markdown
* [COST-5015](https://issues.redhat.com/browse/COST-5015) Migrations to add azure disk capacities
```

### CODE REVIEW

**Review by lcouzens — DISMISSED** [review]

File: koku/reporting_common/migrations/0041_diskcapacity.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            name="DiskCapacity",
+            fields=[
+                ("product_substring", models.CharField(max_length=20, primary_key=True, serialize=False)),
+                ("capacity", models.IntegerField()),
+                (
+                    "provider_type",
+                    models.CharField(
+                        choices=[
+                            ("AWS", "AWS"),
+                            ("OCP", "OCP"),
```
Comment (maskarb) [inline koku/reporting_common/migrations/0041_diskcapacity.py:73]:
```suggestion
```

File: koku/reporting_common/migrations/0041_diskcapacity.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                            ("Azure", "Azure"),
+                            ("GCP", "GCP"),
+                            ("IBM", "IBM"),
+                            ("OCI", "OCI"),
+                            ("AWS-local", "AWS-local"),
+                            ("Azure-local", "Azure-local"),
+                            ("GCP-local", "GCP-local"),
+                            ("IBM-local", "IBM-local"),
+                            ("OCI-local", "OCI-local"),
```
Comment (maskarb) [inline koku/reporting_common/migrations/0041_diskcapacity.py:82]: Do we need these? We might be better off stripping the `-local` during any joins instead.

**Review by maskarb — APPROVED** [review]

**Comment by myersCody** [comment]
One smoke failure definitely unrelated: https://ci.ext.devshift.net/job/koku-pipeline-pr-check-main/2228/

**Pull request merged**